scope & closures/ch1.md

```diff
@@ -241,7 +241,7 @@ Você resolve as referências LHS e RHS através da busca no andar atual, e, se
 
 ## Erros
 
-Por que é relevante se temos uma LHS ou RHS?
+Por que é relevante termos uma LHS ou RHS?
```

Comment:
Essa tradução estava incorreta -- a [frase original](https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch1.md#errors) é "Why does it matter if we call it LHS or RHS?"

Sugiro "Qual a importância de nos referirmos como LHS ou RHS?'